html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html" method="post">
+      <div><label for="email">Email</label></div>
+      <input id="email" type="text" name="email">
+      </form>
+
+    <form action="index.html" method="post">
+      <div>Why is coding cool?</div>
```

Comment:
This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.